scripts/consolidate-links.js

```diff
@@ -24,20 +32,20 @@ const consolidateLinks = () => {
         fs.mkdirSync(path.join(cygwinFolder, "_links"));
     }
 
-    const deleteFile = (file) => {
+    const deleteFile = async (file) => {
         const fileToDelete = path.join(cygwinFolder, path.normalize(file.trim()));
         console.log(`Deleting: ${fileToDelete}`);
         if (!fs.existsSync(fileToDelete)) {
```

Comment:
`existsSync` here: can be replace with async version

Reply:
Thanks @andreypopp ! Good point.